[2022-06-02 23:43:17,469] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-02T23:43:10.666585+00:00 [queued]>
[2022-06-02 23:43:17,477] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-02T23:43:10.666585+00:00 [queued]>
[2022-06-02 23:43:17,478] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 23:43:17,478] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-02 23:43:17,479] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 23:43:17,493] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-02 23:43:10.666585+00:00
[2022-06-02 23:43:17,498] {standard_task_runner.py:52} INFO - Started process 2022 to run task
[2022-06-02 23:43:17,502] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-02T23:43:10.666585+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmp99fb8yq2', '--error-file', '/tmp/tmpx2kmu425']
[2022-06-02 23:43:17,503] {standard_task_runner.py:80} INFO - Job 110: Subtask processing_data
[2022-06-02 23:43:17,560] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-02T23:43:10.666585+00:00 [running]> on host 8e84f5cf889f
[2022-06-02 23:43:17,636] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-02T23:43:10.666585+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-02T23:43:10.666585+00:00
[2022-06-02 23:43:18,541] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-02 23:43:18,543] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-02 23:43:20,353] {python.py:173} INFO - Done. Returned value was: None
[2022-06-02 23:43:20,372] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220602T234310, start_date=20220602T234317, end_date=20220602T234320
[2022-06-02 23:43:20,408] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-02 23:43:20,457] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
